frontend/components/Dashboard/DashboardScreen/index.js

```diff
@@ -90,38 +91,46 @@ const DashboardScreen = (props) => {
 	return (
 		<>
 			<ScrollView style={styles.container} contentContainerStyle={styles.container}>
-				<RMSTitle />
-				<GroupSelection
-					items={GROUPS.map((group) => group.toUpperCase())}
-					onSelectedChange={newGroupHandler}
-				/>
-				<ListsSection
-					categories={CATEGORIES}
-					lists={LISTS}
-					onNewSelection={newListHandler}
-				/>
-				<Text style={styles.title}>Checked Out</Text>
-				{CHECKED_OUT.map((item) => {
-					return (
-						<DashListElement
-							key={item.id}
-							iconName={item.icon}
-							primaryText={item.title}
-							secondaryText={'Return ' + item.returnBy}
-						/>
-					);
-				})}
-				<Text style={styles.title}>Reservations</Text>
-				{RESERVATIONS.map((item) => {
-					return (
-						<DashListElement
-							key={item.id}
-							iconName={item.icon}
-							primaryText={item.title}
-							secondaryText={'Reserved ' + item.reserved}
-						/>
-					);
-				})}
+				<View style={{marginBottom: 100}}>
```

Comment:
Let's avoid injecting styles for good practice. Even if it's just one CSS attribute, make a new StyleSheet object in `styles`